tailwind.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
Same issue here... with the missing empty line at the end of the file (Prettier again to the rescue! :) )